ts/components/messages/MessageDetailData.tsx

```diff
@@ -106,16 +108,15 @@ class MessageDetailData extends React.PureComponent<Props> {
         )}
 
         {this.data.service_name.isSome() && this.data.service_detail.isSome() && (
-          <Text>
+          <H4
+            color="bluegrey"
+            weight="Regular"
+            accessibilityRole="link"
+            onPress={this.props.goToServiceDetail}
+          >
             {`${I18n.t("messageDetails.service")} `}
-            <Text
-              bold={true}
-              link={true}
-              onPress={this.props.goToServiceDetail}
-            >
-              {this.data.service_detail.value.service_name}
-            </Text>
-          </Text>
+            <Link>{this.data.service_detail.value.service_name}</Link>
```

Comment:
`Link` already has `accessibilityRole` set to `link` when `onPress` property is defined
In your implementation all block is clickable 
![a](https://user-images.githubusercontent.com/822471/126602291-62cbdf2c-2821-43a3-88e0-b1be147ef667.png)
But we would to have that only the link is clickable
![b](https://user-images.githubusercontent.com/822471/126602375-934e9a20-6d76-46ef-9524-2f1d62021761.png)

what about this code? 
```typescript
  <Text>
    {`${I18n.t("messageDetails.service")} `}
    <Link onPress={this.props.goToServiceDetail}>
      {this.data.service_detail.value.service_name}
    </Link>
  </Text>
```

If we use h4 only for a row (text for others) we have a minimal miss-alignment

https://user-images.githubusercontent.com/822471/126603395-d8aa6395-6760-421a-a7fb-3a8748a48c77.mov

Reply:
I updated the previous comment since I better understand what happens when the screen reader reads this block

Reply:
I update the previous comment since I better understand what happens when the screen reader reads this block

Reply:
For the tap on all the text, I opted to follow the same behavior that was chosen here https://github.com/pagopa/io-app/pull/3185 in the comments.
My initial idea was to make the whole block a pressable link if the screen reader is enabled, then to write less code it was decided to keep the same behavior. We should choose a unique behavior, which in your opinion would be the most correct one?